README.md

```diff
@@ -1,2 +1,49 @@
 # migration
 Migration tools for TiKV, e.g. online bulk load.
+
+## Build
```

Comment:
should this and the following content be put in the `README.md` file of `online-bulk-load`?

Reply:
updated